package Exceptions.Catching_and_Throwing_Exceptions;

/*--------------------------------------------- there's no point catching an exception ---------------------------------

          unless your code can do something sensible with it
          if we don't catch the exception it will automatically propagate until it will be handled or program crashed



 */